docs/chapter4.md

```diff
@@ -106,10 +98,8 @@ However, if we can define these notions better, it seems we won't need any new n
 Thus, we will arbitrarily decide that the problem description is complete, and move on to the problem specification.
 
 ## 4.2 Stage 2: Specification
-{:#s0015}
-{:.h1hd}
 
-At this point we have an idea-admittedly vague-of what it means to solve a problem in GPS`.` We can refine these notions into representations that are closer to Lisp as follows:
+At this point we have an idea-admittedly vague-of what it means to solve a problem in `GPS`. We can refine these notions into representations that are closer to Lisp as follows:
```

Comment:
I think we can stick to GPS instead of `GPS`.
```suggestion
At this point we have an idea-admittedly vague-of what it means to solve a problem in GPS. We can refine these notions into representations that are closer to Lisp as follows:
```